sphinxcontrib/chapeldomain/README.rst

```diff
@@ -0,0 +1,13 @@
+`chapel.py` is a module that is a local snapshot of the  Pygments highlighter
```

Comment:
Does this file get rendered anywhere via Sphinx? If not, might make sense to make it markdown. If so, we should use double tics:

```suggestion
``chapel.py`` is a module that is a local snapshot of the  Pygments highlighter
```